crates/diesel_models/remove_id.patch

```diff
@@ -0,0 +1,108 @@
+     use crate::enums::diesel_exports::*;
+ 
+     address (address_id) {
+-        id -> Nullable<Int4>,
+         #[max_length = 64]
+         address_id -> Varchar,
+         #[max_length = 128]
+@@ -129,7 +128,6 @@ diesel::table! {
+     use crate::enums::diesel_exports::*;
+ 
+     blocklist (merchant_id, fingerprint_id) {
+-        id -> Int4,
+         #[max_length = 64]
+         merchant_id -> Varchar,
+         #[max_length = 64]
+@@ -284,7 +282,6 @@ diesel::table! {
+     use crate::enums::diesel_exports::*;
+ 
+     customers (customer_id, merchant_id) {
+-        id -> Int4,
+         #[max_length = 64]
+         customer_id -> Varchar,
+         #[max_length = 64]
+@@ -337,7 +334,6 @@ diesel::table! {
+     use crate::enums::diesel_exports::*;
+ 
+     dispute (dispute_id) {
+-        id -> Int4,
+         #[max_length = 64]
+         dispute_id -> Varchar,
+         #[max_length = 255]
+@@ -588,7 +584,6 @@ diesel::table! {
+     use crate::enums::diesel_exports::*;
+ 
+     mandate (mandate_id) {
+-        id -> Int4,
+         #[max_length = 64]
+         mandate_id -> Varchar,
+         #[max_length = 64]
+@@ -634,7 +629,6 @@ diesel::table! {
+     use crate::enums::diesel_exports::*;
+ 
+     merchant_account (merchant_id) {
+-        id -> Int4,
+         #[max_length = 64]
+         merchant_id -> Varchar,
+         #[max_length = 255]
+@@ -678,7 +672,6 @@ diesel::table! {
+     use crate::enums::diesel_exports::*;
+ 
+     merchant_connector_account (merchant_connector_id) {
+-        id -> Int4,
+         #[max_length = 64]
+         merchant_id -> Varchar,
+         #[max_length = 64]
+@@ -741,7 +734,6 @@ diesel::table! {
+     use crate::enums::diesel_exports::*;
+ 
+     payment_attempt (attempt_id, merchant_id) {
+-        id -> Nullable<Int4>,
+         #[max_length = 64]
+         payment_id -> Varchar,
+         #[max_length = 64]
+@@ -832,7 +824,6 @@ diesel::table! {
+     use crate::enums::diesel_exports::*;
+ 
+     payment_intent (payment_id, merchant_id) {
+-        id -> Nullable<Int4>,
+         #[max_length = 64]
+         payment_id -> Varchar,
+         #[max_length = 64]
+@@ -935,7 +926,6 @@ diesel::table! {
+     use crate::enums::diesel_exports::*;
+ 
+     payment_methods (payment_method_id) {
+-        id -> Int4,
+         #[max_length = 64]
+         customer_id -> Varchar,
+         #[max_length = 64]
+@@ -1100,7 +1090,6 @@ diesel::table! {
+     use crate::enums::diesel_exports::*;
+ 
+     refund (merchant_id, refund_id) {
+-        id -> Int4,
+         #[max_length = 64]
+         internal_reference_id -> Varchar,
+         #[max_length = 64]
+@@ -1169,7 +1158,6 @@ diesel::table! {
+     use crate::enums::diesel_exports::*;
+ 
+     roles (role_id) {
+-        id -> Int4,
+         #[max_length = 64]
+         role_name -> Varchar,
+         #[max_length = 64]
+@@ -1276,7 +1263,6 @@ diesel::table! {
+     use crate::enums::diesel_exports::*;
+ 
+     users (user_id) {
+-        id -> Int4,
```

Comment:
@ThisIsMani This id is not used anywhere right?

Reply:
No, we are not.